outerMonster.js

```diff
@@ -0,0 +1,20 @@
+
+const outerMonsterMovement = (outerMonster, gameBoard) => {
+  const temp = {
+    x: outerMonster.x,
+    y: outerMonster.y
+  };
+
+  outerMonster.x += Number(outerMonster.xd);
+  outerMonster.y += Number(outerMonster.yd);
```

Comment:
Mi van akkor, ha a szörny a pálya szélén áll, és hozzáadunk vagy kivonunk egyet a koordinátájából? Nem kapunk array out of bounds-t? Vagy ez máshol van lekezelve?

Reply:
12-17 sorokban az if-nél van lekezelve, hogy ha a pálya szélén van akkor változtassa meg az irányt. ki console.log-oltam, működött. csináljam máshogy?